### DISCUSSION TITLE
Suggestion for an addon

### DISCUSSION THREAD

**Discussion by PigsalotTheFirst** [ScratchAddons/ScratchAddons#4443] [Ideas]
maybe you should add a chat, so like Private chats so we can talk without commenting and everyone seeing it
Suggestion 2 I think you should add custom blocks that help people with scratch addons, like Already made custom blocks that will appear as custom blocks  to somone who doesnt have the addons

you can start by looking at a custom block i made
![block_3_27_2022-12_29_26 PM](https://user-images.githubusercontent.com/102450314/160292748-ed89975c-8794-433f-aff0-5c2ec86de18d.png)
![Screenshot 2022-03-26 12 32 00 PM](https://user-images.githubusercontent.com/102450314/160292786-98d213ef-2055-4be9-bab5-b84fe0053e73.png)

**Comment by Dhuls-Scratch** [comment]
We probably should not any new SA-only blocks. They would only work for those who have SA, which is far from a lot of people (72k on a website used by millions) and would be pretty useless.
Debugger blocks are an only exception

Private chats would be too much work

**Comment by kccuber-scratch** [comment]
This defeats the entire purpose of moderation and reporting.
